recipe/meta.yaml

```diff
@@ -1,27 +1,33 @@
 {% set name = "ruptures" %}
-{% set version = "1.0.6" %}
+{% set version = "1.1.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: aa3eb52e8cee7f2f30eca8210cdd61983dc5b5e41a649034cc52abda319be60b
+  sha256: fb6eda550d749d63d0ea1582f37d290fffa5fd6dc61513d971ef3429681d5255
 
 build:
   number: 0
+  skip: True  # [py<36]
```

Comment:
This is not needed, we skip Python <3.6 automatically.

Reply:
thanks for the quick reply. Resolved in c25ae3f.